packages/react-devtools-shared/src/backend/console.js

```diff
@@ -209,7 +229,31 @@ export function patch({
           debugger;
         }
 
-        originalMethod(...args);
+        if (isInStrictMode) {
+          if (!consoleSettingsRef.hideConsoleLogsInStrictMode) {
+            if (isNode) {
+              originalMethod('\x1b[2m%s\x1b[0m', format(...args));
+            } else {
+              let color;
+              switch (method) {
+                case 'warn':
+                  color = 'rgba(250, 180, 50, .5)';
```

Comment:
I just picked some colors that looked good in the console. We can make them constants. I didn't because we only use them in one place so I didn't add another indirection but I'm not too opinionated either way. 😀